README.md

```diff
@@ -2,6 +2,43 @@
 
 A simple generator plugin for having refinery members-only pages.
 
+## Requirements
```

Comment:
The Readme needs to be clarified:
* `login_member_path` is not an example, it's mandatory.
* need example to use the `previous_url`
* this only works with Devise, so the examples for `after_sign_in_path_for` and `after_sign_out_path_for` are also mandatory for the application to work correctly